x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/common/validation/SourceDestValidator.java

```diff
@@ -419,6 +426,46 @@ public void validate(Context context, ActionListener<Context> listener) {
         }
     }
 
+    public static class RemoteClusterMinimumVersionValidation implements SourceDestValidation {
+
+        private final Version minExpectedVersion;
+
+        public RemoteClusterMinimumVersionValidation(Version minExpectedVersion) {
+            this.minExpectedVersion = minExpectedVersion;
+        }
+
```

Comment:
It would be very helpful if this included a `why`. As a user if all I get is a "remote cluster expected to be version X" without any direction as to why, I would be frustrated.

Reply:
Done.